### ISSUE TITLE
Document bucket strucure on minio

### ISSUE THREAD

**Issue by knl88** [SeaBee-no/documentation#16]
**STORAGE**

Lorand@Sigma2 has now confirmed that we have the granted 10TB storage available for SeaBee.

Next steps is to:

a) Decide/confirm folder structure for data being uploaded (and to be migrated from temporary SharePoint);
> Question is whether we should introduce a 'top-level' based on who (i.e. which institution/organization) is responsible for the respective mission (e.g. folders for NINA, NTNU, IMR, NIVA, Spectrofly, GRID, and perhaps "External"?) and distribute/segregate data accordingly... 

=> Action point: post question and/or call for short dedicated meeting aimed at concluding on this as soon as possible.

b) Based on the above: 

=> Establish/set up and verify relevant methods for uploading data from missions (ref. Rclone, MiniIO app, other?) 

=> Migrate existing data from temporary storage (SeaBee SharePoint) to Sigma.

=> Establish/update user guidance and documentation for those uploading data from missions.

=> (Other steps/actions?)

_Originally posted by @KristofferKa in https://github.com/SeaBee-no/seabee-iac/issues/9#issuecomment-1332366706_

**Comment by knl88** [comment]
Also relevant for #11, but currently minio have access to the `/data` subpath so that application data starts on root

**Comment by knl88** [comment]
@deviirmr How was the structure you envisioned for dmc? Do you want to take point a? I would perhaps think a bit more granular setup like on `seabirds` or project would be better than one big bucket on organization, but I don't have any strong meaning on the topic:)

**Comment by knl88** [comment]
Also think https://github.com/SeaBee-no/documentation/issues/13 is relevant for both a) and b) , more specifically this [section](https://documentation.sigma2.no/files_storage/nird/migration.html#what-can-you-do-as-a-project-leader)

For the first point on b)  I have added a couple of short notes in [storage.md](https://github.com/SeaBee-no/documentation/blob/main/storage.md)

**Comment by JamesSample** [comment]
@knl88 When you get chance, could you provide an overview of the steps to setup RClone with Sigma2 [here](https://seabee-no.github.io/documentation/storage.html#sec-shell), please? Arnt-Børre was asking about how to connect using RClone at a meeting yesterday, and I said we'd update the docs. Obviously credentials etc. should be provided separately, but a short guide describing the setup would be useful.

Thanks!

**Comment by knl88** [comment]
> @knl88 When you get chance, could you provide an overview of the steps to setup RClone with Sigma2 [here](https://seabee-no.github.io/documentation/storage.html#sec-shell), please? Arnt-Børre was asking about how to connect using RClone at a meeting yesterday, and I said we'd update the docs. Obviously credentials etc. should be provided separately, but a short guide describing the setup would be useful.
> 
> Thanks!

Yep, updated the section in ee9aaef, okey? For some minio related tasks `mc` can also be a good option.

**Comment by JamesSample** [comment]
Perfect - thanks Kim!

I'll leave the open as a reminder for me to make some other edits later, but that looks great for RClone :-)
